src/platform/pdf/templates/blue_button_report.js

```diff
@@ -59,6 +59,7 @@ const config = {
   },
   text: {
     boldFont: 'SourceSansPro-Bold',
+    monospaceFont: 'RobotoMono-Regular',
```

Comment:
It would be cool to see a test case added to confirm the new template behavior if possible.

Reply:
I agree. I'll see if I can put together a test for this.